website/docs/docs/cloud/about-cloud/about-dbt-cloud.md

```diff
@@ -4,6 +4,8 @@ id: "dbt-cloud-features"
 sidebar_label: "dbt Cloud features"
 description: "Explore dbt Cloud's features and learn why dbt Cloud is the fastest way to deploy dbt"
 hide_table_of_contents: true
+pagination_next: "docs/cloud/about-cloud/architecture"
```

Comment:
@matthewshaver are you updating this file to 'about-dbt-cloud'? did you get any link errors? i think this page is linked in other locations?

Reply:
The link itself isn't changing, just pointing this page to that location since it's the next in line